DSharpPlus/DiscordConfiguration.cs

```diff
@@ -102,8 +103,8 @@ public string Token
 
         /// <summary>
         /// <para>Sets the level of compression for WebSocket traffic.</para>
-        /// <para>Disabling this option will increase the amount of traffic sent via WebSocket. Setting <see cref="GatewayCompressionLevel.Payload"/> will enable compression for READY and GUILD_CREATE payloads. Setting <see cref="GatewayCompressionLevel.Stream"/> will enable compression for the entire WebSocket stream, drastically reducing amount of traffic.</para>
-        /// <para>Defaults to <see cref="GatewayCompressionLevel.Stream"/>.</para>
+        /// <para>Disabling this option will increase the amount of traffic sent via WebSocket. Setting <see cref="GatewayCompressionLevel.Payload"/> will enable compression for READY and GUILD_CREATE payloads. Setting <see cref="Stream"/> will enable compression for the entire WebSocket stream, drastically reducing amount of traffic.</para>
```

Comment:
Why did u removed GatewayCompressionLevel.

Reply:
Artifact of git merge

Reply:
or was it me

Reply:
No idea. It's been rectified now.